modules/tinymce/src/themes/silver/main/ts/backstage/Backstage.ts

```diff
@@ -58,7 +59,23 @@ const init = (lazySinks: { popup: () => Result<AlloyComponent, string>; dialog:
     translate: I18n.translate,
     isDisabled: () => editor.mode.isReadOnly() || !editor.ui.isEnabled(),
     getOption: editor.options.get,
-    tooltips: TooltipsBackstage(lazySinks.dialog)
+    tooltips: TooltipsBackstage(lazySinks.dialog),
+    checkButtonContext: (specContext: string) => {
+      const resolveContext = () => Strings.contains(specContext, ':') ? specContext.split(':') : [ specContext ];
```

Comment:
Split already does this for you 😉 maybe get rid of the function?
```suggestion
      const resolveContext = () => specContext.split(':');
```